core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -590,15 +590,19 @@ export class BaseUser {
 
   /**
    * This function takes a screenshot of the page.
-   * If there's no parameter for newPage, it checks this.page instead of newPage.
    * If there's no image with the given filename, it stores the screenshot with the given filename in the folder:
-   *  prod_desktop_screenshots or prod_mobile_screenshots for screenshots in production mode
-   *  dev_desktop_screenshots or dev_mobile_screenshots for screenshots in development mode
-   * Otherwise, it compares the screenshot with the image named as the given string to check if they match.
-   * If they don't match, it generates an image in the folder __diff_output__ to show the difference.
+   *   - prod-desktop-screenshots or prod-mobile-screenshots for screenshots in production mode
+   *   - dev-desktop-screenshots or dev-mobile-screenshots for screenshots in development mode
+   * Otherwise, it compares the screenshot with the image named as the given string
+   * to check if they match. If they don't match, it generates an image in the folder
+   * diff-snapshots to show the difference. To check the folder on CI, download the artifact folder
```

Comment:
This is to address the [general comment](https://github.com/oppia/oppia/pull/20945#issuecomment-2458586056), right? If so ... I guess it's fine to put it here, but devs aren't going to see it.

Instead, could you have these instructions get printed out to the bottom fo the CI logs on test failure? That is where developers will look.

Thanks!

Reply:
Done.